src/engraving/dom/undo.cpp

```diff
@@ -2235,6 +2235,9 @@ void InsertRemoveMeasures::insertMeasures()
         for (Staff* staff : score->staves()) {
             Fraction tickStart = fm->tick();
             Fraction tickEnd = lm->endTick();
+            if (tickEnd == tickStart) {
```

Comment:
Looks like this check (and the creation of `tickStart` and `tickEnd`) could just as well happen outside the loop